saleor/graphql/order/mutations/order_fulfill.py

```diff
@@ -177,10 +178,10 @@ def check_total_quantity_of_items(cls, quantities_for_lines):
 
     @classmethod
     def clean_input(cls, info, order, data):
-        site_settings = info.context.site.settings
+        site = load_site(info.context)
```

Comment:
Same here. We should pass the `site` as an argument, we already have the `site` object in perform_mutation.

Reply:
Does not matter. Loader makes only one db hit per request and uses cache for consecutive loads.

Reply:
Yep, but we still need to get this data from the cache. I agree that when the method has plenty of attributes already or we fetched the object like 4 steps above, it's not worth it. But if we have the object in the method that is calling the `clean_input` IMO, it's worth providing it as a method argument.

Reply:
Fixed